WordPress/src/main/java/org/wordpress/android/ui/news/NewsService.kt

```diff
@@ -0,0 +1,13 @@
+package org.wordpress.android.ui.news
+
+import android.arch.lifecycle.MutableLiveData
+import org.wordpress.android.models.news.NewsItem
+
+interface NewsService {
+    fun getNewsItem(): MutableLiveData<NewsItem?>
```

Comment:
I'm not sure `get` is the best method prefix when returning a live data object. Maybe something like `newsItemData()` or `newsItemSource()`. It should also be returning `LiveData` instead of `MutableLiveData` unless you expect someone else to be able to post values.

Reply:
👍